ingest/indexer/service/indexer_streaming_service.go

```diff
@@ -132,7 +132,7 @@ func (s *indexerStreamingService) publishBlock(ctx context.Context, req types.Re
 // as the amount in the event is the amount AFTER the spread factor is applied.
 // therefore, we need to adjust the amount by the spread factor to get the amount BEFORE the spread factor is applied.
 // NOTE: This applies to CL pools only
-func (s *indexerStreamingService) adjustTokenInAmountBySpreadFactor(ctx context.Context, event *types.Event) error {
+func (s *indexerStreamingService) AdjustTokenInAmountBySpreadFactor(ctx context.Context, event *types.Event) error {
```

Comment:
Why do we export this method?

If only for tests, we can export it via `export_test.go` pattern (please see other file examples).

Keeping these unexported is helpful for the information-hiding principle, reducing the cognitive overhead for developers from having too many APIs exposed.

Reply:
Good suggestion. Done.